packages/cdk8s-cli/tsconfig.json

```diff
@@ -28,8 +28,7 @@
     "**/*.ts"
   ],
   "exclude": [
-    "node_modules",
-    "templates/**/*.ts"
```

Comment:
Why?

Reply:
Ah seems JSII re-generated this when I was doing some manual testing. Will revert.